google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentRunConfigurationEditor.java

```diff
@@ -214,7 +217,11 @@ public File get() {
     versionOverrideCheckBox.addItemListener(
         new CustomFieldOverrideListener(versionOverrideCheckBox, versionIdField));
 
-    environmentLabel.setText(environment.localizedLabel());
+    if (environment.isStandard()) {
+      environmentLabel.setText(environment.localizedLabel());
+    } else {
+      environmentLabel.setText(getEnvironmentDisplayableLabel());
+    }
```

Comment:
I was mentioning the conditional on line 128

```java
if (environment.isFlexible()) {
  appEngineCostWarningLabel.setText(
```

I don't think I like the '?' operator here.